src/commands/show.ts

```diff
@@ -1,7 +1,9 @@
 import { Marinade, MarinadeUtils, BN } from '@marinade.finance/marinade-ts-sdk'
+import {StakeInfo} from "@marinade.finance/marinade-ts-sdk/dist/marinade-state/borsh/stake-info";
```

Comment:
is this correct? I think the imports should work without the path: e.g.:
`import {StakeInfo} from "@marinade.finance/marinade-ts-sdk"`
or even 
`import {StakeInfo} from "marinade-ts-sdk"`
What happens if you do that ?

Reply:
We'd have to (and we should) export this class directly from SDK's `./src/index.ts` - otherwise, this is the only way to import it.